packages/components/src/input/src/Input.css

```diff
@@ -182,17 +184,22 @@
 .o-ui-number-input:not(.o-ui-input-disabled):focus-within,
 .o-ui-text-area:not(.o-ui-input-disabled).o-ui-input-has-focus textarea,
 .o-ui-input-focus:not(.o-ui-input-disabled):not(.o-ui-has-button-disabled) {
-    box-shadow: var(--o-ui-focus-ring-inset-md) !important;
+    box-shadow: 0 0 0 0.12rem var(--hop-neutral-surface), 0 0 0 0.219rem var(--hop-primary-border-focus) !important;
```

Comment:
I could be wrong, but it probably doesn't need !important.

Reply:
There's most likely a reason, I hate those but in this case it was probably necessary, as it was already there I would not sweat it.